chrono_common/StoryChunk.h

```diff
@@ -3,6 +3,9 @@
 
 #include <map>
 #include <iostream>
+#include <thallium/serialization/stl/string.hpp>
+#include <thallium/serialization/stl/tuple.hpp>
+#include <thallium/serialization/stl/map.hpp>
```

Comment:
thallium/serialization/stl files are already included  in the develop version of StoryChunk.h.  lines 6-8 can be removed .